kak-lsp.toml

```diff
@@ -159,15 +160,31 @@ args = [
     "--history-file=no",
     "-e",
     """
-        using LanguageServer;
-        using Pkg;
-        import StaticLint;
-        import SymbolServer;
-        import REPL;
-        env_path = dirname(Pkg.Types.Context().env.project_file);
-        server = LanguageServer.LanguageServerInstance(stdin, stdout, env_path, "");
-        server.runlinter = true;
-        run(server);
+    ls_install_path = joinpath(get(DEPOT_PATH, 1, joinpath(homedir(), ".julia")), "environments", "kak-lsp");
+    pushfirst!(LOAD_PATH, ls_install_path);
+    using LanguageServer;
+    popfirst!(LOAD_PATH);
+    depot_path = get(ENV, "JULIA_DEPOT_PATH", "");
+    buffer_file = ENV["kak_buffile"];
+    project_path = let 
+      dirname(something(
```

Comment:
please consider squashing your changes (I don't care about your past mistakes) and documenting the behavior changes (+ rationale) in the commit message.

Reply:
Okay :+1: